### PR TITLE
Remove `build_script` & `test_script`

### PR DESCRIPTION
**Pull request by AyodeAwe**
Removes all `build_script` and `test_script` input references.

See https://github.com/rapidsai/shared-workflows/pull/191#issue-2184593787

Depends on these PRs:

- https://github.com/rapidsai/cudf/pull/15301
- https://github.com/rapidsai/wholegraph/pull/147
- https://github.com/rapidsai/cugraph-ops/pull/615
- https://github.com/rapidsai/cuml/pull/5802
- https://github.com/rapidsai/ucxx/pull/208
- https://github.com/rapidsai/integration/pull/705
- https://github.com/rapidsai/legate-cuml/pull/11
- https://github.com/rapidsai/cudf-private/pull/172
- https://github.com/rapidsai/pynvjitlink/pull/67
- https://github.com/rapidsai/cudf-pandas-integration/pull/101
- https://github.com/rapidsai/gputreeshap/pull/44

### CODE REVIEW

**Comment by bdice** [comment]
@AyodeAwe Please hold off on merging this until all of the above PRs have been forward-merged to 24.06, to avoid breaking 24.06 workflows.

**Review by raydouglass — APPROVED** [review]

**Comment by AyodeAwe** [comment]
All pending forward-merge PRs have been merged so I will now merge this PR.

**Pull request merged**
